src/instructlab/eval/mmlu.py

```diff
@@ -231,9 +238,17 @@ def __init__(
         few_shots: int = 5,
         batch_size: Optional[Union[int, str]] = "auto",
         device: str = ("cuda" if torch.cuda.is_available() else "cpu"),
+        system_prompt: Optional[str] = None,
     ) -> None:
         super().__init__(
-            model_path, None, tasks, model_dtype, few_shots, batch_size, device
+            model_path,
+            None,
+            tasks,
+            model_dtype,
+            few_shots,
+            batch_size,
+            device,
+            system_prompt=system_prompt,
```

Comment:
Just to double check @RobotSail `simple_evaluate` has system_instruction parameter?

Reply:
Looking at this https://github.com/EleutherAI/lm-evaluation-harness/blob/main/lm_eval/evaluator.py#L66 it looks like he got it right

Reply:
Yeah, @abhi1092 we've tested this already in a production system and it does get the system prompt with this.